phases/1-programming.md

```diff
@@ -7,8 +7,31 @@ Read these books and add a 100 - 200 word review to your weekly journal. This sh
 
 * [Getting Real][], 37signals
 * [Passionate Programmer][], Chad Fowler
+* [Pragmatic Programmer][],  Andrew Hunt & Dave Thomas
 
 #### **Training:**
+* Week 0: CLI & Command Line
+* Week 1: Git at Sparkbox
+  * ♫ "Shunting trucks and hauling freight" ♫
+* Week 2: Functional Programming
+  * Breaking a Problem Down by Breaking a Problem Down
+  * Reason Your Way to Fame and Fortune
+* Week 3: Object Oriented Programming
+  * "Mr. Watson, come here. I want to see you." --A. Bell
+  * Cells: The Building Blocks of Life!
```

Comment:
Conway’s?

Reply:
Yes, and reinforcing data hiding and encapsulation. I want to look at how message passing works in biology.